backend/DataClasses.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#DataClasses.py
+#----------------------------------
+# Created By : Matthew Kastl
+#----------------------------------
+""" This file contains a collection of data classes for use by Flare.
+These data classes are to make function call more generic and readable
+ """ 
+#----------------------------------
+# 
+#
+#Imports
+
+
+class Call():
```

Comment:
what is this class for? no documentation. I'm assuming.... it's for the series names for the different data sources and post processing and stuff, but it might be worth writing that down

Reply:
Documented